disnake/webhook/sync.py

```diff
@@ -505,6 +511,8 @@ def delete(self, *, delay: Optional[float] = None) -> None:
             The message was deleted already.
         HTTPException
             Deleting the message failed.
+        WebhookTokenMissing
```

Comment:
`versionchanged` missing in parts of this file

Reply:
I thought I got all of these...

Reply:
https://github.com/DisnakeDev/disnake/pull/471/commits/332e4d1c318caec58b47b46d04f0512c56f6f4df